### PR TITLE
Changes emag price back to 6TC

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

Reverts #1415
we arent prebase anymore 4head

#### Changelog

:cl:  
tweak: Cryptographic Sequencer now costs 6TC once more
/:cl:

### CODE REVIEW

**Comment by Amelia0010** [comment]
why

**Comment by ynot01** [comment]
> why

emag is a really fun and useful item but it's not very play-defining and has majorly mediocre use apart from a few uses (emagging borg namely), it encourages care-free play and personally ive had a lot more fun in my traitor rounds just memeing with it, emagging a bunch of doors just to piss off sec and ai; not really powerful but fun and has some utility

emags also encourage friendly non-edgy-assassination-i-work-alone play where you can offer to emag someones machines (cargo console, chem dispenser, etc) in exchange for services (letting you use them without blabbing, money, info)

overall emag is a _fun_ item rather than an **op** one, and i think it should be made more readily available to those who want to fit that playstyle

if certain items were priced specifically to counter emag combo then they can be raised on request (e.g. revolver) but honestly you can just kill anyone in your path with an item like revolver, and an emag doesnt really matter at that point

**Comment by SomeguyManperson** [comment]
tl;dr

**Comment by ynot01** [comment]
> tl;dr

emeg fun and not op :))

**Comment by Szyszkrzyneczka** [comment]
Hello i upvote because i think this is good change even though i didn't play tator in like two months

**Comment by jcatjmeow** [comment]
Emag is quite strong. It has infinite uses, and the applications are vast and useful. I dun like dis pr

**Comment by Szyszkrzyneczka** [comment]
give it like 20 uses and no random crits, instant balance

**Comment by ThatLing** [comment]
> emag is a really fun and useful item  

so lets buff it?

>but it's not very play-defining and has majorly mediocre use apart from a few uses (emagging borg namely), 

it has a lot of uses, just search 'emag_act' in the codebase

>it encourages care-free play

so let's buff it?

>not really powerful but fun and has some utility

being able to open any door or locker you like, order crates with more syndicate gear, taking over borgs is not powerful?

> emags also encourage friendly non-edgy-assassination-i-work-alone play where you can offer to 
emag someones machines (cargo console, chem dispenser, etc) in exchange for services (letting you use them without blabbing, money, info)

sounds like helping an enemy of state which is a capital crime

> overall emag is a _fun_ item rather than an **op** one

see above

> and i think it should be made more readily available to those who want to fit that playstyle

see above, you want to buff an op item

> if certain items were priced specifically to counter emag combo then they can be raised on request (e.g. revolver) but honestly you can just kill anyone in your path with an item like revolver, and an emag doesnt really matter at that point

can't kill a door with a revolver

**Comment by SomeguyManperson** [comment]
>can't kill a door with a revolver

you fool

**Comment by ThatLing** [comment]
> 
> 
> > can't kill a door with a revolver
> 
> you fool

have fun with your 0 ammo afterwards

**Comment by ynot01** [comment]
i think its clear that i think emag is far less than op

**Review by alexkar598 — APPROVED** [review]

**Comment by ThatLing** [comment]
> 
> 
> i think its clear that i think emag is far less than op

i think its clear that the emag is indeed op

**Comment by ynot01** [comment]
> > > can't kill a door with a revolver
> > 
> > 
> > you fool
> 
> have fun with your 0 ammo afterwards

imagine not filling your backpack with speedloaders

**Comment by ThatLing** [comment]
> 
> 
> > > > can't kill a door with a revolver
> > > 
> > > 
> > > you fool
> > 
> > 
> > have fun with your 0 ammo afterwards
> 
> imagine not filling your backpack with speedloaders

imagine spending all of your TC on ammo just to fail your steal objective because you couldn't open a locker in time

**Comment by ynot01** [comment]
> > > > > can't kill a door with a revolver
> > > > 
> > > > 
> > > > you fool
> > > 
> > > 
> > > have fun with your 0 ammo afterwards
> > 
> > 
> > imagine not filling your backpack with speedloaders
> 
> imagine spending all of your TC on ammo just to fail your steal objective because you couldn't open a locker in time

you fool

```
/datum/design/a357
	name = "Speed Loader (.357)"
	id = "a357"
	build_type = AUTOLATHE
	materials = list(MAT_METAL = 30000)
	build_path = /obj/item/ammo_box/a357
	category = list("hacked", "Security")
```

**Comment by ThatLing** [comment]
> 
> 
> > > > > > can't kill a door with a revolver
> > > > > 
> > > > > 
> > > > > you fool
> > > > 
> > > > 
> > > > have fun with your 0 ammo afterwards
> > > 
> > > 
> > > imagine not filling your backpack with speedloaders
> > 
> > 
> > imagine spending all of your TC on ammo just to fail your steal objective because you couldn't open a locker in time
> 
> you fool
> 
> ```
> /datum/design/a357
> 	name = "Speed Loader (.357)"
> 	id = "a357"
> 	build_type = AUTOLATHE
> 	materials = list(MAT_METAL = 30000)
> 	build_path = /obj/item/ammo_box/a357
> 	category = list("hacked", "Security")
> ```

but how am i supposed to get a security lathe without access

**Comment by ynot01** [comment]
> > > > > > > can't kill a door with a revolver
> > > > > > 
> > > > > > 
> > > > > > you fool
> > > > > 
> > > > > 
> > > > > have fun with your 0 ammo afterwards
> > > > 
> > > > 
> > > > imagine not filling your backpack with speedloaders
> > > 
> > > 
> > > imagine spending all of your TC on ammo just to fail your steal objective because you couldn't open a locker in time
> > 
> > 
> > you fool
> > ```
> > /datum/design/a357
> > 	name = "Speed Loader (.357)"
> > 	id = "a357"
> > 	build_type = AUTOLATHE
> > 	materials = list(MAT_METAL = 30000)
> > 	build_path = /obj/item/ammo_box/a357
> > 	category = list("hacked", "Security")
> > ```
> 
> but how am i supposed to get a security lathe without access

thas not the sec lathe thats hacked autolathe

**Comment by ThatLing** [comment]
> 
> 
> > > > > > > > can't kill a door with a revolver
> > > > > > > 
> > > > > > > 
> > > > > > > you fool
> > > > > > 
> > > > > > 
> > > > > > have fun with your 0 ammo afterwards
> > > > > 
> > > > > 
> > > > > imagine not filling your backpack with speedloaders
> > > > 
> > > > 
> > > > imagine spending all of your TC on ammo just to fail your steal objective because you couldn't open a locker in time
> > > 
> > > 
> > > you fool
> > > ```
> > > /datum/design/a357
> > > 	name = "Speed Loader (.357)"
> > > 	id = "a357"
> > > 	build_type = AUTOLATHE
> > > 	materials = list(MAT_METAL = 30000)
> > > 	build_path = /obj/item/ammo_box/a357
> > > 	category = list("hacked", "Security")
> > > ```
> > 
> > 
> > but how am i supposed to get a security lathe without access
> 
> thas not the sec lathe thats hacked autolathe

it literally says security

**Comment by ynot01** [comment]
> > > > > > > > > can't kill a door with a revolver
> > > > > > > > 
> > > > > > > > 
> > > > > > > > you fool
> > > > > > > 
> > > > > > > 
> > > > > > > have fun with your 0 ammo afterwards
> > > > > > 
> > > > > > 
> > > > > > imagine not filling your backpack with speedloaders
> > > > > 
> > > > > 
> > > > > imagine spending all of your TC on ammo just to fail your steal objective because you couldn't open a locker in time
> > > > 
> > > > 
> > > > you fool
> > > > ```
> > > > /datum/design/a357
> > > > 	name = "Speed Loader (.357)"
> > > > 	id = "a357"
> > > > 	build_type = AUTOLATHE
> > > > 	materials = list(MAT_METAL = 30000)
> > > > 	build_path = /obj/item/ammo_box/a357
> > > > 	category = list("hacked", "Security")
> > > > ```
> > > 
> > > 
> > > but how am i supposed to get a security lathe without access
> > 
> > 
> > thas not the sec lathe thats hacked autolathe
> 
> it literally says security

yes its under the security section of autolathe when it is hacked do u play this game 🤔

**Comment by CMOisLing** [comment]
Are you guys seriously arguing about using revolvers to open doors and lockers?

**Comment by ynot01** [comment]
what else would we be arguing about on an emag pr

**Comment by Oakboscage** [comment]
Making it a tad cheaper won't kill anyone and it's fun to use.

**Comment by Moja10** [comment]
Emag is strong as shit, but 8 tc is nearly half your TC and that makes its investment dubious compared to what you could get for a similar price, or the number of things you cannot get because of an investment in an emag. For example you can't get a revolver + emag, big bomb + emag,, desword + emag, essentially it being priced at 8 TC means that you cannot invest into certain very important items and a lot of weapon and survival items because of its high cost, and its high cost does not offer any significant defense or way to fight aside from emagging borgs or emagging doors to escape. In addition, the steep cost means as well you need to invest in your safety, things like storage implants, adrenal implants, and freedom implants so you don't just get arrested and have your round ended. Emag's price makes it niche because of how the steep price cuts off a lot of other options that are arguably necessary for a traitor's survival. 

With this being said though, it's not the end of the world if it remains 8 tc, the price doesn't make it worthless by any means, but I think it would be a fine change to make it 6 TC.
